src/Game/Game.cpp

```diff
@@ -19,7 +19,7 @@
 
 // C++ standard includes
 #include <sstream>
-#include <string>
+#include <string.h>
```

Comment:
There should be SDL_setenv (which actually just calls setenv), but judging by setenv manpage it'll work without duplicating string. So yeah, we could use that (SDL_setenv version, as it deals with platform-specific env) and drop strdup.